tests/test_event_sequences.py

```diff
@@ -0,0 +1,11 @@
+import altair as alt
+
+from eds_scikit.datasets import load_event_sequences
+from eds_scikit.plot.event_sequences import plot_event_sequences
+
+df_events = load_event_sequences()
+
+
+def test_visit_merging():
```

Comment:
We are almost done with this review! :) A last thing to focus on is to extend the test to the if / else of the function to have good code coverage. You should set the optional arguments and perform the same check you wrote.

Reply:
It should be better now ! How/Where can I check the % of code coverage for this PR ?

Reply:
This is done automatically in the CI by codecov when this draft becomes a PR, you can also run codecov locally with `pytest --cov=eds_scikit`